addons/website/models/website.py

```diff
@@ -654,6 +656,8 @@ def _render_snippet(key):
                 for key in generated_content:
                     if response.get(key):
                         generated_content[key] = (name_replace_parser.sub(website.name, response[key], 0))
+                gpt_process_time = time.time() - start_time
+                logger.info("GPT process time: %s seconds", gpt_process_time)
```

Comment:
I would be interested to know if we have a analysis tool to monitor this on the saas :thinking:

Reply:
Not that I know of.
We can just ask for some logs here and there but we don't have any access to the logs ourselves, let alone some monitoring tools. At least from what I know.
Could be a thing tho, the runbot team has some graph about perfs based on test logs AFAIK.